packages/components/src/components/text-field/text-field.tsx

```diff
@@ -96,6 +96,8 @@ export class TextField {
   /** (optional) Injected CSS styles */
   @Prop() styles?: string;
 
+  /** (optional) Makes type `input` behave as a controlled component in React */
+  @Prop() controlled?: boolean = false;
```

Comment:
so i replaced (optional) with (experimental) but I'm not sure if that is what you meant or to change the prop name to something like `experimental_controlled` ? Wasn't sure because in other components like the dropdown there is also a `controlled` prop but without any experimental prefix

Reply:
yes, haha, I was thinking about `experimental_controlled` — the old Dropdown is going to be deprecated.

Would it work with the snake_case? 🤔